build/crunchy-postgres-exporter/Dockerfile

```diff
@@ -1,31 +1,28 @@
-ARG BASEOS
-ARG BASEVER
-ARG PREFIX
-FROM ${PREFIX}/pgo-base:${BASEOS}-${BASEVER}
+# DL3006: The latest version of "ubi-micro" is fine.
+# hadolint ignore=DL3006
```

Comment:
When I run `hadolint --ignore=DL3006 Dockerfile` locally, I get:
``` 
Dockerfile:10 DL3020 error: Use COPY instead of ADD for files and folders
Dockerfile:11 DL3020 error: Use COPY instead of ADD for files and folders
Dockerfile:12 SC2035 info: Use ./*glob* or -- *glob* so names with dashes won't become options.
Dockerfile:14 DL3041 warning: Specify version with `dnf install -y <package>-<version>`.
Dockerfile:24 DL3020 error: Use COPY instead of ADD for files and folders
Dockerfile:25 DL3020 error: Use COPY instead of ADD for files and folders
Dockerfile:26 DL3020 error: Use COPY instead of ADD for files and folders
```